tensorflow/core/profiler/internal/gpu/BUILD

```diff
@@ -138,6 +140,24 @@ tf_cuda_library(
     ],
 )
 
+tf_cuda_library(
+    name = "rocm_tracer",
+    srcs = if_rocm(["rocm_tracer.cc"]),
+    hdrs = if_rocm(["rocm_tracer.h"]),
+    copts = tf_profiler_copts() + tf_copts(),
+    visibility = ["//visibility:public"],
```

Comment:
The visibility can be restricted to the targets that actually use it. I don't think this needs to be visible globally.

Reply:
the visibility setting is based on the same for the CUDA side, which is also public.